### PR TITLE
fix: Modify global styles to remove host

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #733 & #734

## Summary

fix: Modify global styles to remove host

- Moves all the global styles into a mixin that is then applied on each component.

### CODE REVIEW

**Comment by driskull** [comment]
@jcfranco @asangma maybe this could be the first step for cleaning up the CSS. Then the second step could remove the header and heading styles from base/type.

File: src/assets/styles/_mixins.scss [hunk 1]
```diff
@@ -1,5 +1,115 @@
 // Mixins
 
+@mixin calciteLightThemeVars {
```
Comment (asangma) [inline src/assets/styles/_mixins.scss]: Sorry if we discussed this, but I am not clear on why these vars are in a mixin. Same with the DarkThemeVars.

Comment (driskull) [reply src/assets/styles/_mixins.scss]: The light vars are stated twice so its in a mixin for that

**Comment by driskull** [comment]
Closing, the other PR is better.

**Pull request closed without merging**